configuration/configuration_test.go

```diff
@@ -64,18 +64,20 @@ var configStruct = Configuration{
 		Prefix string `yaml:"prefix,omitempty"`
 		Secret string `yaml:"secret,omitempty"`
 		TLS    struct {
-			Certificate string   `yaml:"certificate,omitempty"`
-			Key         string   `yaml:"key,omitempty"`
-			ClientCAs   []string `yaml:"clientcas,omitempty"`
+			Certificate    string   `yaml:"certificate,omitempty"`
+			Key            string   `yaml:"key,omitempty"`
+			CACertificates []string `yaml:"cacertificates,omitempty"`
+			ClientCAs      []string `yaml:"clientcas,omitempty"`
 		} `yaml:"tls,omitempty"`
 		Debug struct {
 			Addr string `yaml:"addr,omitempty"`
 		} `yaml:"debug,omitempty"`
 	}{
 		TLS: struct {
-			Certificate string   `yaml:"certificate,omitempty"`
-			Key         string   `yaml:"key,omitempty"`
-			ClientCAs   []string `yaml:"clientcas,omitempty"`
+			Certificate    string   `yaml:"certificate,omitempty"`
+			Key            string   `yaml:"key,omitempty"`
+			CACertificates []string `yaml:"cacertificates,omitempty"`
```

Comment:
To be clear, this should be called `RootCAs` to match up with the `crypto/tls` config.